Client/Frontend/Browser/Browser.swift

```diff
@@ -170,11 +170,12 @@ class Browser: NSObject {
             }
 
             if ErrorPageHelper.isErrorPageURL(url) {
-                let decodedURL = ErrorPageHelper.decodeURL(url)
-                if !AboutUtils.isAboutURL(decodedURL) {
-                    return decodedURL
-                } else {
-                    return nil
+                if let decodedURL = ErrorPageHelper.decodeURL(url) {
+                    if !AboutUtils.isAboutURL(decodedURL) && !ErrorPageHelper.isErrorPageURL(decodedURL) {
```

Comment:
The `isErrorPageURL` shouldn't be necessary. `ErrorPageHelper.decodeURL(url)` recursively decodes the URL until it's no longer an error page.